controllers/drcluster_controller.go

```diff
@@ -277,6 +272,46 @@ func filterDRClusterMCV(mcv *viewv1beta1.ManagedClusterView) []ctrl.Request {
 	}
 }
 
+func filterDRClusterSecret(ctx context.Context, reader client.Reader, secret *corev1.Secret) []ctrl.Request {
+	log := ctrl.Log.WithName("filterDRClusterSecret").WithName("Secret")
+
+	drcusters := &ramen.DRClusterList{}
```

Comment:
(nit) drClusters ~ the name is missing `l`.

Reply:
Right, thank you. Copy-paste from old code.  Changed